introspection-engine/introspection-engine-tests/tests/add_prisma1_defaults/mod.rs

```diff
@@ -22,13 +22,19 @@ async fn add_cuid_default(api: &TestApi) -> crate::TestResult {
         })
         .await?;
 
-    let dm = indoc! {r#"
-        model Book {
-            id  String @id @default(cuid())
-        }
-    "#};
+    let native_type = if sql_family.is_postgres() {
+        "@db.VarChar(25)"
+    } else {
+        "@db.Char(25)"
+    };
 
-    assert_eq_datamodels!(dm, &api.introspect().await?);
+    let dm = formatdoc! {r#"
```

Comment:
Thought from discussion with Tom: having a barrel extension where type::p_string has our default mappings for native types for the Prisma type string. This way we avoid having to sprinkle native types everywhere when that is not the test focus.